packages/auth-sdk/src/types/event.ts

```diff
@@ -0,0 +1,4 @@
+export enum Event {
+  CredentialsUpdated = 'credentialsrupdated',
```

Comment:
Shouldn't it be like this?
```suggestion
  CredentialsUpdated = 'credentialsupdated',
```